### ISSUE TITLE
[PARENT TICKET for NATIVE AUTH] End to End Native Auth on Explorer and Data DEX

### ISSUE THREAD

**Issue by newbreedofgeek** [Itheum/data-dex#1006]
This is parent ticket that captures the main feature we need to ship for 1.5.

Once this ticket is done, @fuzzyduke can QA this based on this UX:

**Test Flow 1:**
- Alice is not logged into Data DEX or Explorer
- Alice logs into Data DEX via a wallet
- Alice go to wallet and clicks on "View Data", the data opens WITHOUT he needed to "sign message" like before. All "View Data" button needs to have the same behaviour (i.e. details page)
- Alice clicks on "Explore" on the Trailblazer Data NFT and is take to the Itheum Explorer
- As she was logged into the Data DEX, she is also now automatically logged into the Explorer (via Native Auth - xPortal hub method in query-string)
- On Explorer, Alice can "View Data" on Apps like Trailblazer and her Wallet and she never needs to "sign message" like before. 

Test Flow 2:
- Alice is not logged into Data DEX or Explorer
- Alice goes to Explorer and login in via a wallet
- She goes to the Trailblazer page. She does not have the Data NFT, so the page says "you need to buy this data nft first". Click on this link she's taken to the details/offers page on the Data DEX
- As she was logged into theExplorer, she is also now automatically logged into the Data DEX (via Native Auth - xPortal hub method in query-string)

Items in this ticket include.
- [backend] https://github.com/Itheum/sdk-mx-data-nft/issues/30
- [backend] https://github.com/Itheum/core-mx-marshal-brontes-service/issues/7 
- [backend] https://github.com/Itheum/core-mx-marshal-brontes-service/issues/10
- [frontend] https://github.com/Itheum/explorer-dapp/issues/128
- [frontend] https://github.com/Itheum/data-dex/issues/1007
- [frontend] https://github.com/Itheum/explorer-dapp/issues/150
- [frontend] https://github.com/Itheum/data-dex/issues/1008
- [docs] https://github.com/Itheum/sdk-mx-data-nft/issues/35

**Comment by damienen** [comment]
This can only be tested once we have merged into "test"

**Comment by damienen** [comment]
This flow works. Other bugs are flagged as separate issues, so closing this
